### DISCUSSION TITLE
Import GnuCash csv

### DISCUSSION THREAD

**Discussion by larrotino** [ananthakumaran/paisa#246] [Q&A]
Hi! i'm trying to move to paisa from Gnucash...but i'm struggling import my tons of gnucash data. can you teach me how?
here a little exaple of the cvs... thanks! :)
[Gnucash_test.csv](https://github.com/user-attachments/files/15906191/Gnucash_test.csv)

**Comment by obsilva** [comment]
Don't know whether you are still looking for assistance, but I did this transition myself just last week. Here is what I did:

  1. In GnuCash, use the `Save As...` option to save your file in `sqlite` format - I originally had it in XML
  2. Use `gnucash2ledger.py` script from [pygnucash](https://github.com/MatzeB/pygnucash) project to convert your transactions
  3. Go through the result and manually make any changes that you require. My adjustments were mostly cosmetics, mainly to match [Paisa account naming convention](https://paisa.fyi/reference/accounts/) and to have the commodities and accounts in their own file etc
